README.md

```diff
@@ -43,11 +43,13 @@ The plugin can be configured in the [**semantic-release** configuration file](ht
 ```
 
 With this example:
-- the commits that contains `BREAKING CHANGE` or `BREAKING CHANGES` in their body will be considered breaking changes (by default the [angular preset](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-angular/index.js#L14) checks only for `BREAKING CHANGE`)
```

Comment:
Why change that? Is that confusing or redundant in any way?

Reply:
I guess not. But, the later points do not have that what is `By default` option for them so, I guess it would not be good to convey that here.

Reply:
On second thought, I would be better to have that Info too. 😅

Reply:
@pvdlg Any updates? IMO: The original message is a lot confusing. Instead having a short crisp message is better.